package.json

```diff
@@ -14,13 +14,15 @@
     "dotenv": "^15.0.0",
     "jsonwebtoken": "^8.5.1",
     "memory-cache": "^0.2.0",
-    "ms": "^2.1.3",
     "next": "12.0.10",
     "react": "17.0.2",
     "react-dom": "17.0.2"
   },
   "devDependencies": {
     "eslint": "8.8.0",
     "eslint-config-next": "12.0.10"
+  },
+  "engines": {
+    "node": ">=0.16"
```

Comment:
is that right? didn't you want to require major 16?

Reply:
This should be requiring above 16. I can add 0.16.16.

Reply:
```suggestion
    "node": ">=16.0"
```